extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/config/runtime/TracesRuntimeConfig.java

```diff
@@ -46,5 +46,5 @@ public interface TracesRuntimeConfig {
      */
```

Comment:
We need to also clarify this documentation or people will mess up the config because we are no longer enforcing a double upfront.
It should read:
````
Sampler argument. Depends on the `quarkus.otel.traces.sampler` property.
Fallbacks to the legacy property <code>quarkus.opentelemetry.tracer.sampler.ratio</code>.
<p>
When setting the stock sampler to `traceidratio` or `parentbased_traceidratio` you need to set a `double` compatible value like `0.01d` or `0.5d`. It is kept as a `String` to allow the flexible customisation of alternative samplers.
<p>
Defaults to `1.0d`.
````

Reply:
Done